### ISSUE TITLE
Invalid Authentication (tested and verified good email/password)

### ISSUE THREAD

**Issue by cdmichaelb** [aijayadams/hass-blueair#15]
![image](https://user-images.githubusercontent.com/1451709/177232039-39b01b69-aefe-4469-8696-0993de000062.png)

**Comment by calebsw83** [comment]
Special characters in your password will prevent you from connecting via HA

**Comment by DavidSimacek** [comment]
Special characters in the password are required by the Blueair password policy.

**Comment by coreydwillis** [comment]
Is there a workaround for this given that Blueair requires special characters and using special characters will fail in HA?

**Comment by stylerw** [comment]
Yeah, this is blocking me too. I've been trying to find a fix here without much luck, so I'm curious if the Dev has any approaches to fix this.
